airbyte-integrations/connectors/destination-google-sheets/destination_google_sheets/buffer.py

```diff
@@ -0,0 +1,124 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from typing import Any, List, Mapping
+
+from airbyte_cdk import AirbyteLogger
+from airbyte_cdk.models import AirbyteStream
+
+
+class WriteBuffer:
+
+    # Default instance of AirbyteLogger
+    logger = AirbyteLogger()
+    # Buffer for input records
+    records_buffer = []
+    # Placeholder for streams metadata
+    stream_info = []
```

Comment:
seems like it better to use dict instead of list, with 'stream_name' as key

Reply:
I agree, done.